packages/atomic/src/templates/default.html

```diff
@@ -1,8 +1,6 @@
 <style>
-  :host {
-    border-top: 1px solid var(--atomic-neutral);
+  :host > .result-root {
```

Comment:
This change is breaking since old templates only work when `display: block` is set on `.result-root`.

Reply:
Like we discussed the default loaded template should be very very simple, just a title or something. We can remove the rest